src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTestFixture.kt

```diff
@@ -29,13 +27,23 @@ import java.nio.file.Paths
 import java.util.concurrent.CompletableFuture
 import java.util.concurrent.TimeUnit
 import java.util.regex.Pattern
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
 
-open class CodyIntegrationTextFixture : BasePlatformTestCase() {
-  private val logger = Logger.getInstance(CodyIntegrationTextFixture::class.java)
+@RunWith(JUnit4::class)
+abstract class CodyIntegrationTestFixture : BasePlatformTestCase() {
 
   override fun setUp() {
     super.setUp()
-    configureFixture()
+    val methodName = name
+    val method =
+        this.javaClass.getMethod(methodName)
+            ?: throw IllegalStateException(
+                "No method with name $methodName found in ${this.javaClass.name}")
+    if (method.isAnnotationPresent(TestFile::class.java)) {
```

Comment:
Maybe we could use the default test file of no annotation is added?

Reply:
Good call; I'll clean that up.

Reply:
Looks like you missed this one @steveyegge

Reply:
Yeah, there really isn't a good choice for a default right now, since the only test file is specific to `DocumentCodeTest`. 

But I went ahead and used that as the default for now, until we get more tests with disk files.

Reply:
I had to roll this change back because it broke all the new tests, which don't have testResources files on disk.